language-support/qute/qute.tmLanguage.json

```diff
@@ -362,6 +362,10 @@
         {
           "match": "\\b(const|goto)\\b",
```

Comment:
This pattern should be removed. const, goto, return etc are not a keyword, etc for Qute template.